filters/auth/tokeninfo_test.go

```diff
@@ -713,3 +743,45 @@ func TestOAuthTokeninfoValidate(t *testing.T) {
 		})
 	}
 }
+
+func TestOAuthTokeninfoValidateArgs(t *testing.T) {
+	tio := TokeninfoOptions{
+		URL:     "https://auth.test",
+		Timeout: testAuthTimeout,
+	}
+	spec := NewOAuthTokeninfoValidate(tio)
+
+	t.Run("valid", func(t *testing.T) {
+		for _, def := range []string{
+			`oauthTokeninfoValidate("{}")`,
+			`oauthTokeninfoValidate("{optOutAnnotations: [oauth.disabled], optOutHosts: [ '^.+[.]domain[.]test$', '^exact[.]test$' ], unauthorizedResponse: 'Authentication required, see https://auth.test/foo'}")`,
```

Comment:
I would've added oauthTokeninfoValidate fiters with single arguments as well, like this:
+ `oauthTokeninfoValidate("{optOutAnnotations: [oauth.disabled], optOutHosts: [ '^.+[.]domain[.]test$', '^exact[.]test$' ]}")`
+ `oauthTokeninfoValidate("{unauthorizedResponse: 'Authentication required, see https://auth.test/foo'}")`

Reply:
This quickly goes out of hands but I added.